assets/app/scripts/controllers/overview.js

```diff
@@ -334,6 +355,11 @@ angular.module('openshiftConsole')
       }
     };
 
+    function isCustomHost(route) {
+      var defaultHost = route.serviceName + '-' + route.metadata.namespace;
```

Comment:
@pweil- any way to know if a route was generated vs custom? That check is attempting to prefer a custom route, over a generated one to show on the overview page, since we only show one (with a "and N more" link to the service page where you can see all of the routes)